contracts/core/Registry.sol

```diff
@@ -258,4 +286,12 @@ contract Registry is AccessControl {
     function _generateIdentityId(uint256 _nonce) internal view returns (bytes32) {
         return keccak256(abi.encodePacked(_nonce, msg.sender));
     }
+
+    /// @notice Transfer thefunds recovered  to the recipient
+    /// @param _token The address of the token to transfer
+    /// @param _recipient The address of the recipient
+    function recoverFunds(address _token, address _recipient) external onlyIdentityOwner(ALLO_OWNER) {
+        uint256 amount = _token == address(0) ? address(this).balance : ERC20(_token).balanceOf(address(this));
+        _transferAmount(_token, _recipient, amount);
+    }
```

Comment:
I don't think this function will work. `onlyIdentityOwner` looks up `identitiesById[_identityId].owner` but there is no `identitiesById` for `ALLO_OWNER`.

I think what we want to do is `require(isMemberOfIdentity(ALLO_OWNER, msg.sender));` since we granted the role, which makes them a member.

Reply:
oh true! good point